.storybook/main.ts

```diff
@@ -0,0 +1,77 @@
+import path from "path"
+
+import type { StorybookConfig } from "@storybook/nextjs"
+import { propNames } from "@chakra-ui/react"
+
+/**
+ * Note regarding package.json settings related to Storybook:
+ *
+ * There is a resolutions option set for the package `jackspeak`. This is related to a
+ * workaround provided to make sure storybook ( as of v7.5.2) works correctly with
+ * Yarn v1
+ *
+ * Reference: https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092
+ *
+ * The primary recommendation is to upgrade to Yarn 3 if possible
+ */
+
+/**
+ * Temporary list of allowed stories during migration to NextJS
+ */
+const tempStoryList = ["../src/components/Buttons/**/*.stories.@(ts|tsx)"]
```

Comment:
Incrementally add components as necessary since there are some that still have gatsby package imports. 

There is more than just the Button files that could be included here. What do you think @pettinarip ?

Reply:
Makes sense @TylerAPfledderer. A similar approach to the one we are taking in the pages' migration.